CHANGELOG.md

```diff
@@ -3,6 +3,27 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.1.4] Preview Release
+### Added
+- Added APIs for DataSourceFactory and OSGI Framework [#700](https://github.com/Microsoft/mssql-jdbc/pull/700)
+- Added support for OffsetDateTime to be passed as 'type' in ResultSet.getObject() [#830](https://github.com/Microsoft/mssql-jdbc/pull/830)
+- Added support for Active Directory MSI Authentication [#838](https://github.com/Microsoft/mssql-jdbc/pull/838)
+- Added more datatype tests to increase code coverage [#878](https://github.com/Microsoft/mssql-jdbc/pull/878)
```

Comment:
Probably reword to something like "Added additional junit testing for datatypes". Don't need to mention coverage.